assets/stylesheets/shared/_color-schemes.scss

```diff
@@ -11,7 +11,7 @@
 	// --color-accent-dark: #{};
 
 	--color-success: #{$alert-green};
-	// --color-success-light: #{};
+	--color-success-light: #{lighten( $alert-green, 20 )};
```

Comment:
@drw158 I used 20 instead of 15 here as it seems that's more commonly used throughout Calypso. Let me know if that'd work for legacy Classic Blue.

Reply:
This seems ok to me!